src/app/api/organizations/[organizationId]/route.ts

```diff
@@ -24,3 +27,34 @@ export async function DELETE(
   }
   return new NextResponse(undefined, { status: 204 });
 }
+
+// @route PUT /api/organizations/[organizationId] - updates an existing organization in the database
+export async function PUT(
+  _request: NextRequest,
```

Comment:
pedantic, but could you remove the `_` from the name? It's a convention that indicates that it is an unused parameter.